src/store/adjustVault.ts

```diff
@@ -266,6 +266,13 @@ export const adjustVaultErrorsAtom = atom(get => {
     collateralError = 'Insufficient funds.';
   }
 
+  if (
+    collateralAction === CollateralAction.Withdraw &&
+    AmountMath.isEmpty(newLocked)
+  ) {
+    collateralError = 'Exceeds available collateral.';
```

Comment:
could there be this error and others concurrently?

Reply:
Multiple error conditions could be true simultaneously, but `collateralError` only holds one. I don't think it's worth the overhead and noisiness of showing multiple errors at once.

Reply:
Agree. It's not clear to me if the error message precedence here is intentional or not. Would be important to know when refactoring, especially since there aren't tests to imply a spec.

Reply:
Yea makes sense, just added some documentation to make it less unclear.